js/w3c/templates/sotd.html

```diff
@@ -131,7 +131,7 @@
             a group
             {{/if}} operating under the
             <a{{#if doRDFa}} id="sotd_patent" property='w3p:patentRules'{{/if}}
-            href='https://www.w3.org/Consortium/Patent-Policy-20040205/'>5 February 2004 W3C Patent
+            href='https://www.w3.org/Consortium/Patent-Policy-20170801/'>5 February 2004 W3C Patent
```

Comment:
Should this use the Patent Policy's full name:

`5 February 2004 (updated 1 August 2017)`

@plehegar to give an authoritative answer.